lib/spree_products_qa/configuration.rb

```diff
@@ -0,0 +1,13 @@
+module SpreeProductsQa
```

Comment:
@mausamp we might want to look at this file for defining this gem related configurations